components/styles/MotionHeader.styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+import { theme } from '../themes/DefaultTheme';
+
+const StyledHeader = styled(motion.header)`
+	max-width: 100%;
+`;
+
+export default function MotionHeader(props) {
+	return (
+		<StyledHeader
+			variants={theme.motion.pageTransitionVariants}
```

Comment:
@tr3mulant I couldn't figure out a way to actually use the motion object stored in the theme without importing it and using it like this. Do you know of a way I can get it from the props?

Reply:
[Is this what we're after?](https://styled-components.com/docs/advanced)
```
import styled, { useTheme } from 'styled-components';
import { motion } from 'framer-motion';

const StyledMain = styled(motion.main)`
	height: 100%;
	max-width: 100%;
`;

export default function MotionMain(props) {
	const theme = useTheme();
	return (
		<StyledMain
			variants={theme.motion.pageTransitionVariants}
			initial='hidden'
			animate='visible'
			exit='exit'
			{...props}
		/>
	);
}
```

Reply:
I'm not sure if this is better than just importing the object but yeah

Reply:
I like useTheme here because while it's true in this instance that I don't need to know if it's light or dark theme in this case that won't always be true.

Reply:
With the new changes to theming, should we just pass in the theme as a prop like the other components that use theme stuff? 
I see in the new version, you're now importing the themeCommon object instead.

Reply:
I'm working on a version that just uses the useTheme() function so we don't have to pass the theme around all over. It seems to be working on the <main> element but not the <header> element, not sure why yet. Hope to get this resolved before our meeting.

Reply:
OK got a fix for this in place.